config/channels/in-memory-channel/README.md

```diff
@@ -1,19 +1,17 @@
 # In-Memory Channels
 
-In-memory channels are a best effort Channel. They should **NOT** be used in
-Production. They are useful for development.
+In-memory channels are a best effort channel. They have the following characterics:
```

Comment:
I don't think you should remove the warning. Even with a dead letter sink, InMemoryChannel is still **not** production worthy. 

It is easily possible for a message to get dropped, if the dispatcher Pod fails for any reason.

Reply:
There are use cases for which dropping messages is acceptable, like for instance IoT. So I think the in-memory channel is production worthy for those use cases. 
I want to remove this warning not to scare developers, and tell them "yes use it by be aware of the limitations".
Does this make sense?

Reply:
I agree with Lionel, I like the addition of the properties table below, it makes it clear that the user is choosing this for the properties provided.